api/v1/db/json/notes-dao.js

```diff
@@ -191,4 +197,22 @@ const patchNoteByID = (noteID, body) => new Promise((resolve, reject) => {
   }
 });
 
-module.exports = { getNotes, postNotes, patchNoteByID };
+const deleteNoteByID = noteID => new Promise((resolve, reject) => {
+  try {
+    const rawNote = fetchNote(noteID);
+    if (!rawNote) {
+      resolve(null);
+    }
+
+    const studentID = noteID.split('-')[0];
```

Comment:
Same as the comment I left in this [PR](https://github.com/osu-mist/notes-api/pull/5#discussion_r243852760), I think we should move this logic to the path file level.